src/lib/y2storage/proposal/devices_planner_strategies/legacy.rb

```diff
@@ -0,0 +1,253 @@
+#!/usr/bin/env ruby
```

Comment:
The class should not be considered a script itself, so we have not to include a shebang.

Reply:
Fine with me. But there's no consistency in the storage-ng code here.